string cam1
string cam2
---
string status